static/js/modules/download.js

```diff
@@ -137,7 +137,7 @@ DownloadItem.prototype.refresh = function() {
     contentType: 'application/json',
     dataType: 'json'
   });
-  this.promise.then(this.handleSuccess.bind(this));
+  this.promise.done(this.handleSuccess.bind(this));
```

Comment:
Sounds good to me!  Should we work in the `fec-style` version bump in this PR to just ensure full testing of the updates?